### ISSUE TITLE
There is an error when I use PySide6 6.4 and QtAwesome - `AttributeError: Cannot reassign members.`

### ISSUE THREAD

**Issue by lizhuoxuan** [spyder-ide/qtawesome#216]
### versions：
- python 3.9.13
- pyside6 6.4
- qtawesome 1.2.0

### code:
```
import sys
from PySide6 import QtCore, QtWidgets, QtGui
import qtawesome

class MyWidget(QtWidgets.QWidget):
    def __init__(self):
        super().__init__()

        self.button = QtWidgets.QPushButton("click here")

        self.layout = QtWidgets.QVBoxLayout(self)
        self.layout.addWidget(self.button)

        self.button.clicked.connect(self.showMessage)

    @QtCore.Slot()
    def showMessage(self):
        msgBox = QtWidgets.QMessageBox()
        msgBox.setText("Hello world")
        msgBox.setStandardButtons(QtWidgets.QMessageBox.Ok)
        ret = msgBox.exec()


if __name__ == "__main__":
    app = QtWidgets.QApplication([])

    widget = MyWidget()
    widget.resize(300, 200)
    widget.show()

    app.setStyle(QtWidgets.QStyleFactory.create("Fusion")) 
    sys.exit(app.exec())
```
### error:
```
Traceback (most recent call last):
  File "D:\work\pyside6_study\ps1.py", line 3, in <module>
    import qtawesome
  File "D:\work\pyside6_study\venv\lib\site-packages\qtawesome\__init__.py", line 23, in <module>
    from qtpy import QtCore, QtWidgets, QtGui
  File "D:\work\pyside6_study\venv\lib\site-packages\qtpy\QtCore.py", line 116, in <module>
    Qt.MidButton = Qt.MouseButton.MiddleButton = Qt.MiddleButton
  File "shibokensupport/enum_310.py", line 529, in __setattr__
AttributeError: Cannot reassign members.
```

**Comment by dalthviz** [comment]
Hi @lizhuoxuan thank you for the feedback! As the traceback points out this is an issue over QtPy (for more info about the related issue, it is available here: https://github.com/spyder-ide/qtpy/issues/373).

Let us know if the info above helps!

**Comment by lizhuoxuan** [comment]
> Hi @lizhuoxuan thank you for the feedback! As the traceback points out this is an issue over QtPy (for more info about the related issue, it is available here: [spyder-ide/qtpy#373](https://github.com/spyder-ide/qtpy/issues/373)).
> 
> Let us know if the info above helps!

Yes,it is very useful.thanks!
